snuba/utils/metrics/backends/testing.py

```diff
@@ -13,12 +13,19 @@ class RecordedMetricCall:
     tags: Tags
 
 
+@dataclass(frozen=True)
+class RecordedEventCall:
+    value: str
```

Comment:
Why have a single value? Can't it actually contain the different fields like `title`, `text`, `alert_type` etc?

Reply:
I had it stored as a json string. Since this is mostly dummy data for testing I didn't bother too much with keeping its structure.